engine/engine/src/engine.cpp

```diff
@@ -1302,6 +1302,8 @@ namespace dmEngine
 
         dmGui::SetDisplayProfiles(engine->m_GuiContext, engine->m_DisplayProfiles);
 
+        dmPlatform::ShowWindow(engine->m_Window);
```

Comment:
To avoid the "flashing", we postpone showing the window until right before we draw something to it.